### DISCUSSION TITLE
如果您有好的项目可以给dtkwidget提交Pr

### DISCUSSION THREAD

**Discussion by Decodetalkers** [linuxdeepin/developer-center#3473] [Developer Discussion | 开发者讨论]
https://github.com/linuxdeepin/dtkwidget

我们会将您在github或者其他代码托管仓库的项目写到readme中，作为展示。

一方面可以帮您的项目被人认识，
(另一方面可以丰富我们的reamde

**Comment by Wing-summer** [comment]
成熟且稳定的：
WingHexExplorer ： https://github.com/Wing-summer/WingHexExplorer
较稳定但不成熟：
WingTool ：https://github.com/Wing-summer/WingTool
实用（原因应急缓解开发）小工具：
ScreenLight：https://github.com/Wing-summer/ScreenLight
（还有一个，太烂了，在我使用过程中有很多已知问题，就不丢人了）

**Reply by Decodetalkers** [reply]
可以给我们Pr，照着dtkwidget的Readme添加就可以了

**Reply by Wing-summer** [reply]
> 可以给我们Pr，照着dtkwidget的Readme添加就可以了

收到，已 PR

**Reply by Decodetalkers** [reply]
> > 可以给我们Pr，照着dtkwidget的Readme添加就可以了
> 
> 收到，已 PR

我给你的Pr了一点建议，改了就可以过Ci了，周一就直接合并了

**Reply by Decodetalkers** [reply]
> > 可以给我们Pr，照着dtkwidget的Readme添加就可以了
> 
> 收到，已 PR

不要气馁。。再提一次吧2333

**Reply by Decodetalkers** [reply]
提了At我下

**Reply by Wing-summer** [reply]
> 提了At我下

完了，我又粗心了，把变更标识 HEAD 之类的提交上去了，我的锅，咋整。

**Reply by Decodetalkers** [reply]
> > 提了At我下
> 
> 完了，我又粗心了，把变更标识 HEAD 之类的提交上去了，我的锅，咋整。

修改，
git commit --amend
直接保存
然后 git push origin master -f

记住要-f

**Reply by Wing-summer** [reply]
> 提了At我下

![image](https://user-images.githubusercontent.com/59953528/197345316-7f18fe89-43ce-4a0d-884f-897d10e34c0b.png)

是这个东西，好像是因为冲突自动加上的，内容还没变化

**Reply by BLumia** [reply]
本地代码中修正，之后再 `git commit -a --amend` :)

**Comment by wineee** [comment]
@SeptemberHX 来不来加一个  :)

**Reply by SeptemberHX** [reply]
🤣 比较遗憾的是我写的都不是 dtk 的，大部分窗口都是 qt 的，deepin 主题让他们看着像 dtk
